README.md

```diff
@@ -58,22 +59,33 @@ var client = new Client("YOUR-XAPI-KEY", Model.Enum.Environment.Test);//or Model
 var checkout = new Checkout(client);
```

Comment:
maybe it will be good to add another method in `paymentRequest`, similar to
`paymentRequest.AddCardData("4111111111111111", "10", "2020", "737", "John Smith");`

but with encrypted credentials, like
`paymentRequest.AddEncryptedCardData("test_4111111111111111", "test_10", "test_2020", "test_737", "John Smith");`
as it's easier to test for merchants without full pci.

Reply:
Yes that would be a good idea. I will create a ticket for it :)